jsk_topic_tools/cmake/nodelet.cmake

```diff
@@ -4,7 +4,8 @@ macro(jsk_nodelet _nodelet_cpp _nodelet_class
     _single_nodelet_exec_name _CODE_PARAM_NAME)
   list(APPEND ${_CODE_PARAM_NAME} ${_nodelet_cpp})
   set(NODELET ${_nodelet_class})
-  set(DEFAULT_NODE_NAME ${_single_nodelet_exec_name})
+  string(RANDOM __NODENAME_PREFIX)
+  set(DEFAULT_NODE_NAME JSK_NODELET_${__NODENAME_PREFIX}_${_single_nodelet_exec_name})
```

Comment:
@k-okada Is there any rationale that you randomized the prefix?
I wonder any target misses caches in the case of `catkin build --force-cmake`.
How about changing `__NODENAME_PREFIX` to, for example, just `PROJECT_NAME`?

Reply:
@furushchev thank you for feedback, yes `PROJECT_NAME` would be better https://github.com/jsk-ros-pkg/jsk_common/pull/1591